pylua/test/websocket_test.py

```diff
@@ -0,0 +1,23 @@
+#!/usr/bin/env python2
+# -*- coding: utf-8 -*-
+
+import thread
+import time
+import lua
+from pylua.load_lua import open_script
+from pylua.network import * #pylint: disable=wildcard-import, unused-wildcard-import
+
+class TestWebsocket(object):
+    def test1(self): #pylint: disable=no-self-use
```

Comment:
@OHerasym, please add some meaningful test names. Test name must be self-descriptive.

Try to not suppress warnings until it's really required. In this case there is common workaround, just rename `self` -> `dummy_self` or something like this(there is some special convention for such cases).